### PR TITLE
Updating code to use device from is mobile() is desktop()/dhruv

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
## Changes:

1. Updated the `isMobile()` and `isDesktop()` functions in the accounts section by implementing the `useDevice` hook wherever it was missed out.

2. Resolved the Height issue in Proof of Identity in tablet view 
**Issue:** 

Screenshot 2024-06-04 at 3 45 29 PM

**Resolved:** 

Screenshot 2024-06-04 at 3 44 15 PM

### CODE REVIEW

File: packages/account/src/Components/icon-message-list/list-item.tsx [hunk 1]
```diff
@@ -2,17 +2,17 @@ import React from 'react';
...
 import { observer, useStore } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
+import { useDevice } from '@deriv-com/ui';
 
 type TListItem = {
     text: React.ReactNode;
     index?: number;
 };
 
 const ListItem = observer(({ text, index }: TListItem) => {
```
Comment (fasihali-deriv) [inline packages/account/src/Components/icon-message-list/list-item.tsx]: Hi @Dhruv-deriv ,

Now as you have already removed `store` so you can remove this `observer` as well .

@amina-deriv please correct me if I am wrong thank you 🙇

File: packages/account/src/Components/leave-confirm/leave-confirm.tsx [hunk 2]
```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { RouteComponentProps, useHistory, withRouter } from 'react-router-dom';
```
Comment (fasihali-deriv) [inline packages/account/src/Components/leave-confirm/leave-confirm.tsx]: Can you please Ignore this file as I am also working on this. 👀 

CC: @amina-deriv

File: packages/account/src/Components/icon-message-list/list-item.tsx (hunk 1 above)
Comment (Dhruv-deriv) [reply packages/account/src/Components/icon-message-list/list-item.tsx]: hey @fasihali-deriv  we are using observer here should we still remove it ?

Screenshot 2024-06-04 at 2 49 12 PM

File: packages/account/src/Components/leave-confirm/leave-confirm.tsx (hunk 2 above)
Comment (Dhruv-deriv) [reply packages/account/src/Components/leave-confirm/leave-confirm.tsx]: i have removed my changes @fasihali-deriv 👍

Comment (fasihali-deriv) [reply packages/account/src/Components/leave-confirm/leave-confirm.tsx]: Thank youo @Dhruv-deriv 🙏

File: packages/account/src/Components/icon-message-list/list-item.tsx (hunk 1 above)
Comment (amina-deriv) [reply packages/account/src/Components/icon-message-list/list-item.tsx]: we can remove observer wrapper

Comment (Dhruv-deriv) [reply packages/account/src/Components/icon-message-list/list-item.tsx]: hey @fasihali-deriv i have removed it as we dont need observer thanks 👍

**Review by utkarsha-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Pull request merged**